Packages/Models/Sources/Models/Status.swift

```diff
@@ -57,12 +57,12 @@ public protocol AnyStatus {
 
 public struct StatusViewId: Hashable {
   let id: String
-  let editedAt: ServerDate?
```

Comment:
I pushed this minor change together with the rest of the PR. I noticed `StatusViewId` was still showing up in the Profile, now because of ARC. I decided to try embedding `Date` directly into the `StatusViewId`, and it no longer does. I'm not sure how to explain it, but it's most like some Swift optimization.